src/app/views/user/user-settings/user-profile/user-profile.component.scss

```diff
@@ -1,7 +1,166 @@
 @import '../../../../styles/variables';
+@import '../../../../styles/buttons';
 
-hr {
-  margin: 35px 0 25px;
-  border: 0;
-  border-top: 1px solid #eeeeee;
+.profile-form-container {
+  padding-top: 50px;
+  max-width: 500px;
+  margin: 0 auto;
+
+  .profile-input-container {
+    padding-bottom: 1.5em;
+    margin-bottom: 15px;
+
+    label {
+      color: black;
+      font-size: 20px;
+    }
+
+    > textarea {
+      overflow: hidden;
+      word-wrap: break-word;
+      resize: none;
+    }
+
+    .profile-input-permit-radio-container {
+      display: flex;
+      margin-top: 30px;
+
+      > div {
+        text-align: center;
+        width: 90px;
+
+        > label {
+          font-size: 16px;
+          opacity: 0.7;
+
+          > span {
```

Comment:
having lots of `>` can make it hard to change/add elements in the future. Could we perhaps replace it with classes? or what do  you think?